srcpkgs/droidcam/files/droidcam.desktop

```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Encoding=UTF-8
+Version=3.2.1
+Name=Droidcam
+Type=Application
+Exec=droidcam
+Icon=droidcam
+Categories=GNOME;AudioVideo;
+Comment=A tool for using your Android phone as a wireless/usb webcam
+
```

Comment:
They have a file, use `vsed -e "s/Icon=.*/Icon=droidcam/" -i droidcam.desktop` to fix it up in `post_build`.